thingif/src/main/java/com/kii/thingif/query/GroupedHistoryStatesQuery.java

```diff
@@ -70,7 +70,7 @@ public Builder setClause(@Nullable QueryClause clause) {
 
         /**
          * Set firmware version for grouped query.
-         * @param firmwareVersion firmware version
+         * @param firmwareVersion older firmware version, which defined this alias.
```

Comment:
@yongpingchen Thank you. I got it.

```
@param firmwareVersion firmware version equals or earlier than a firmware version which defined the states
```

I think this more clearly describes your intention. What do you think?